### ISSUE TITLE
Feedback from google

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1595]
the first 'name' is invalid, the rest 'could be better'

![grafik](https://github.com/jemproject/JEM-Project/assets/2960236/11811837-78de-4690-91b9-78bde2fa784c)

how can we add this?

**Comment by Heklaterriol** [comment]
see  [#1334 ](https://github.com/jemproject/JEM-Project/issues/1334)

**Comment by Heklaterriol** [comment]
@jojo12 What did you test?

Some testing and wild thought from my side: 

I validated several views with
https://search.google.com/test/rich-results and 
https://validator.schema.org/ 

In simple list e.g. it looks like in this example:
```
@type | Event
startDate | 2023-06-24T16:30:00
endDate | 2023-06-26T22:00:00
name | Balkan Beatz
url | https://j4.datenablage.info/event/2:balkan-beatz
identifier | https://j4.datenablage.info/event/2:balkan-beatz
location |  
@type | Place
name | Kamikaze
address |  
@type | PostalAddress
name | Freiburg im Breisgau,
```

I encountered no errors but some not critical warnings like:
- Field "eventStatus" missing (optional)
- Field "organizer" missing (optional)
- Field "performer" missing (optional)
- Field "offers" missing (optional)
- Field "endDate" missing (optional)
- Field "image" missing (optional)
- Field "eventAttendanceMode" missing (optional)

`endDate` is only missing when there is no end date in the event.
Possible solution: if no `end date` we set for the microdata `end date` = `start date`.

`eventStatus` can be set to a valid value

and `image` is set if available, maybe we use a default image for the microdata if there is not image set fo the event

**module**
there is no microdata in the modules. Should we add it there?

The specifications for microdata in events: https://schema.org/Event

**Comment by Heklaterriol** [comment]
let's continue this issue on #1334

**Comment by Heklaterriol** [comment]
can be closed [#1334](https://github.com/jemproject/JEM-Project/issues/1334)
